### PR TITLE
Add tip and overtime income reform

### LINKED ISSUE #45: Add tip and overtime income reform
We should do it separately due to the nature of the reform

### PR DESCRIPTION
**Pull request by PavelMakarchuk**
Fixes #45

### CODE REVIEW

File: ui_components.py [hunk 1]
```diff
@@ -55,11 +55,46 @@ def render_personal_info():
...
+        max_value=500000, 
+        value=0, 
+        step=500, 
+        format="$%d"
+    )
+    
+    tip_income = st.slider(
+        "Annual tip income",
+        min_value=0,
+        max_value=100000,
```
Comment (MaxGhenis) [inline ui_components.py:80]: remove sliders

File: results.py [hunk 2]
```diff
@@ -78,19 +105,47 @@ def calculate_values(categories, simulation, year):
             result_dict[category] = 0
     return result_dict
 
-def calculate_consolidated_results(reform_name, state, is_married, child_ages, income, social_security_retirement,
-                                head_age, spouse_age=None, medical_expenses=0, real_estate_taxes=0,
-                                interest_expense=0, charitable_cash=0, charitable_non_cash=0,
-                                qualified_business_income=0, casualty_loss=0):
+def calculate_consolidated_results(reform_name, state, is_married, child_ages, income,
```
Comment (MaxGhenis) [inline results.py:108]: apply `black` to format

File: results.py [hunk 3]
```diff
@@ -78,19 +105,47 @@ def calculate_values(categories, simulation, year):
...
+
+    # Create situation dictionary with reform name
     situation = create_situation(
-        state, is_married, child_ages, income, social_security_retirement,
-        head_age, spouse_age, medical_expenses, real_estate_taxes, interest_expense,
-        charitable_cash, charitable_non_cash, qualified_business_income,
-        casualty_loss
+        state=state,
+        is_married=is_married,
+        child_ages=child_ages,
```
Comment (MaxGhenis) [inline results.py:134]:
```suggestion
        child_ages=child_ages,  # This is a list.
```

File: results.py [hunk 4]
```diff
@@ -78,19 +105,47 @@ def calculate_values(categories, simulation, year):
...
+def calculate_consolidated_results(reform_name, state, is_married, child_ages, income, 
+                                social_security_retirement, head_age, spouse_age=None, 
+                                medical_expenses=0, real_estate_taxes=0, interest_expense=0, 
+                                charitable_cash=0, charitable_non_cash=0, qualified_business_income=0, 
+                                casualty_loss=0, tip_income=0, overtime_income=0):
     """
     Calculates metrics for a single reform with detailed breakdowns.
     """
-    # Create situation dictionary
+    # For baseline, add all income to regular employment income
```
Comment (MaxGhenis) [inline results.py:116]: skip by applying the structure to baseline

**Review by MaxGhenis — CHANGES_REQUESTED** [review]
also check the args to format_fig to move the logo up a bit

File: results.py (hunk 4 above)
Comment (PavelMakarchuk) [reply results.py]: Applying a different method by adjusting the US package to accommodate the reform inputs

**Pull request merged**